src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
```

Comment:
The import statement for 'PostInfo' should be changed to 'import { PostInfo } from '../PostInfo/PostInfo';' to correctly import the component from its own folder.

Reply:
We use an index.ts file in each component folder witch have export-import inside 'export * from './PostInfo';' in purpose to shorten import path. This method is offered  in MA video lessons.